Source/WebCore/rendering/RenderBox.h

```diff
@@ -436,6 +436,22 @@ class RenderBox : public RenderBoxModelObject {
     void updateLogicalHeight();
     virtual LogicalExtentComputedValues computeLogicalHeight(LayoutUnit logicalHeight, LayoutUnit logicalTop) const;
 
+    // Finds the nearested multiple of blockStepSize to marginBoxBlockSize
+    static LayoutUnit computeBlockStepSizeRoundedUp(LayoutUnit marginBoxBlockSize, LayoutUnit blockStepSize)
```

Comment:
are you planning on using it in other compilation unit?

Reply:
Not necessarily. Do you think this type of method should not be a static member or that it may need to be changed in the future?

Reply:
I think this could just be in the .cpp and not on the class at all.

Reply:
Fair. Changed.